dvc/scm/git/backend/pygit2.py

```diff
@@ -234,3 +234,39 @@ def status(
         self, ignored: bool = False
     ) -> Tuple[Mapping[str, Iterable[str]], Iterable[str], Iterable[str]]:
         raise NotImplementedError
+
+    def merge(
+        self,
+        rev: str,
+        commit: bool = True,
+        msg: Optional[str] = None,
+        squash: bool = False,
+    ) -> Optional[str]:
+        # pylint: disable=no-name-in-module
+        from pygit2 import GIT_RESET_MIXED, GitError
```

Comment:
pylint doesn't detect module level C imports, not sure if this is normal for pylint or if it's specific to  the way pygit2 builds/includes libgit2 exports?

Reply:
You can add the pygit2 here in the config:
https://github.com/iterative/dvc/blob/a2c91d20581df2e402a020a5887a72fdee7bc73a/.pylintrc#L6